src/components/ENavButton.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <a class="ENavButton">
+    <slot />
+  </a>
+</template>
+
+<style scoped>
+.ENavButton {
+  display: inline-flex;
+  padding: var(--p-1);
+  border: var(--border-DEFAULT) solid;
+  font-family: var(--font-family-mono);
+  font-size: var(--text-xs);
+  text-transform: uppercase;
+  color: var(--gray-300);
+}
+.ENavButton:not(:first-child) {
+  margin-left: calc(var(--border-DEFAULT) * -1);
+}
+.ENavButton:hover {
+  border-image: url("../assets/bg-texture-xs.gif") 1;
```

Comment:
pildifailid kaivad `/public/` kataloogi ja urlid algavad `/` (review pr'is tehtud)

Reply:
ega see pilt palju borderi seest valja ei paista aga las ta olla as placeholder. ilmselt teeb sinna custom elavad taustad edaspidi.